cloud/envs/gcp.py

```diff
@@ -159,8 +159,8 @@ def __init__(self, instance):
       m = re.search(r'(\d+\.\d+)\.\d+', tf.__version__)
       self.tf_version = m.group(1)
       if self.tf_version[0] == "2":
```

Comment:
Can you test and see if just removing this conditional works now? We should add a conditional that handles the case when we're running a nightly build instead of just looking for a "2" and then assuming nightly

Reply:
Do you know if there's a way to differentiate between tf2.x and nightly builds? I'm running `tf-nightly==2.2.0.dev20200303` but when I do a tf.version it shows up as 2.2, so far as as cloud is concerned, they're the same thing.

Reply:
Think I got it, we're going to assume that if the version includes "dev" it'll be nightly.